src/shallow.ts

```diff
@@ -1,26 +1,37 @@
-export default function shallow<T, U>(objA: T, objB: U) {
-  if (Object.is(objA, objB)) {
-    return true
+const shallow = <A, B>(_a: A, _b: B) => {
```

Comment:
I'm not sure how shallow equals is required here, shouldn't `shallow({ a: undefined }, {})` return `true`? The previous version would return `false` (afaict) but this newer version would return `true`

Reply:
good question. I believe it's a subtle change, and if we clearly write it in the release note, we can change the behavior.

Reply:
Yeah, I think the newer behavior is better.